api/scripts/download_from_s3.py

```diff
@@ -0,0 +1,68 @@
+import csv
+import datetime
+import os
+import sys
+import asyncio
+import logging
+import boto3
+import json
+from databases import Database
+
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+import asyncclick as click
+
+from notd.store.schema import TokenCollectionsTable, TokenMetadataTable
+from notd.store.schema_conversions import collection_from_row
+from core.util import date_util
+from core.s3_manager import S3Manager
+
+
+@click.command()
+@click.option('-s', '--start-id-number', 'startId', required=True, type=int)
+@click.option('-e', '--end-id-number', 'endId', required=True, type=int)
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=100)
+async def download_from_s3(startId: int, endId: int, batchSize: int):
```

Comment:
please rename this something better (including the file). Something like `calculate_all_token_fields`